src/components/Energiatodistus/energiatodistukset.svelte

```diff
@@ -400,9 +400,9 @@
       <span class="material-icons">attachment</span>
       &nbsp;
       <Link
-        text={i18n('energiatodistus.lataa-xlsx')}
-        href={'/api/private/energiatodistukset/xlsx/energiatodistukset.xlsx' +
-          queryStringForXlsx} />
+        text={i18n('energiatodistus.lataa-csv')}
+        href={'/api/private/energiatodistukset/csv/energiatodistukset.csv' +
+          queryStringForCsv} />
```

Comment:
Oli vanhastaankin, mutta näitä voisi jatkossa kirjoittaa backtick stringeinä. Muuttuisi siis muotoon
`\`/api/private/energiatodistuset/csv/energiatodistukset.csv${queryStringForCsv}\``

Reply:
:+1: